test/functional/test_framework/test_node.py

```diff
@@ -34,7 +34,8 @@
 )
 
 BITCOIND_PROC_WAIT_TIMEOUT = 60
-
+VALGRIND_VERSION_MAJOR = 3
+VALGRIND_VERSION_MINOR = 14
```

Comment:
The scope can be narrowed for these and the meaning could be clarified by doing it like this on L103 instead :)

```
MIN_VALGRIND_VERSION_EARLY_EXIT = (3, 14)
if valgrind_version and valgrind_version >= MIN_VALGRIND_VERSION_EARLY_EXIT:
```